webextensions/manifest/commands.json

```
@@ -255,16 +264,21 @@
               },
               "edge": "mirror",
               "firefox": {
-                "version_added": "48"
+                "version_added": "48",
+                "notes": "Available for use in Manifest V2 only."
```

Sorry, again, page action available in Firefox for MV3
```suggestion
                "version_added": "48"
```
Also created https://github.com/mdn/content/issues/34363 to fix documentation re-Safari